src/seedu/addressbook/ui/TextUi.java

```diff
@@ -91,28 +79,28 @@ public String getUserCommand() {
     public void showWelcomeMessage(String version, String storageFilePath) {
         String storageFileInfo = String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
         showToUser(
-                DIVIDER,
-                DIVIDER,
+                Formatter.DIVIDER,
+                Formatter.DIVIDER,
                 MESSAGE_WELCOME,
                 version,
                 MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
                 storageFileInfo,
-                DIVIDER);
+                Formatter.DIVIDER);
     }
 
     public void showGoodbyeMessage() {
-        showToUser(MESSAGE_GOODBYE, DIVIDER, DIVIDER);
+        showToUser(MESSAGE_GOODBYE, Formatter.DIVIDER, Formatter.DIVIDER);
```

Comment:
This method seems to be formatting the message too.
Could `showToUser(formatter.formattedGoodByeMsg())` be better?